translations/ui-licenses/en.json

```diff
@@ -85,6 +85,7 @@
   "supplementaryDocs.none": "No supplementary documents have been added",
 
   "terms.add": "Add term",
+  "terms.setValue": "A value must be set for this term in order to save a note",
```

Comment:
```suggestion
  "terms.setValue": "A value must be set for this term in order to save a note.",
```

Reply:
Actually, can you move this to a key like `errors.noteWithoutValue` as well? Trying to keep all the error messages together.

Reply:
done